crates/ruff_workspace/src/configuration.rs

```diff
@@ -583,6 +585,29 @@ impl Configuration {
                 .chain(selection.unfixable.iter())
                 .chain(selection.extend_fixable.iter())
             {
+                #[allow(deprecated)]
+                if matches!(selector, RuleSelector::Nursery) {
+                    let suggestion = if preview.is_disabled() {
+                        "Use the `--preview` flag instead."
+                    } else {
+                        "Use the `PREVIEW` selector instead."
+                    };
+                    warn_user_once!("The `NURSERY` selector has been deprecated. {}", suggestion);
```

Comment:
Does `warn_user_once!("The `NURSERY` selector has been deprecated. {suggestion}")` work?

Reply:
Probably yeah I didn't realize it supported that at first and then did later :)

Reply:
```suggestion
                    warn_user_once!("The `NURSERY` selector has been deprecated. {suggestion}");
```